### DISCUSSION TITLE
How to remove horizontal green lines on scrollbar caused by ideavim

### DISCUSSION THREAD

**Discussion by fmuhic** [JetBrains/ideavim#540] [Q&A] [answered]
![Screenshot from 2022-08-31 10-05-23](https://user-images.githubusercontent.com/28514114/187630087-9515ff1a-a04e-40f8-a47c-a37d6cd75794.png)
Does anyone know how to remove these green lines from scrollbar. I know its caused by this plugin because when I turn it off they disappear.

**Comment by AlexPl292** [comment, marked as answer]
We actually don't generate any kind of such lines. Maybe you can hover one of them and you'll get more info what exactly they are referring to?

**Reply by citizenmatt** [reply]
I think it might be search result highlights. It looks like you've got a search that is matching the very start of the line. Search for something else - `/foo` - or use `:nohl` to clear the current results.

**Reply by fmuhic** [reply]
Now I feel very silly
